gtk4/Gir.toml

```
@@ -1659,7 +1659,8 @@ trust_return_value_nullability = false
 
 [[object]]
 name = "Gtk.PrintCapabilities"
-status = "manual" # to cfg_condition on linux see gtk-rs.com/gir/issues1081
+status = "generate" # to cfg_condition on linux see gtk-rs.com/gir/issues1081
```

```suggestion
status = "generate"
```
the comment is no longer needed btw

---

Oh right!